docs/manual.rst

```diff
@@ -213,6 +213,36 @@ The resume data format is very similar to the .torrent file format, and when
 including the info-dict in the resume data, the resume file can be used as a
 .torrent file (with just a few minor exceptions).
 
+BitTorrent v2 torrents
+======================
+
+BitTorrent v2 introduces a number of features outlined in `this blog post`_ as
+well as `BEP 52`_. The v2 protocol introduces the possibility to use a merkle
+hash tree instead of a flat list of piece hashes. It also supports *hybrid* torrents,
+that are both valid classing torrents (v1) as well as valid v2 torrents. Hybrid
+torrents contain both a flat list of piece hashes as well as a merkle hash tree.
+
+.. _`this blog post`: https://blog.libtorrent.org/2020/09/bittorrent-v2/
+.. _`BEP 52`: https://www.bittorrent.org/beps/bep_0052.html
+
+This introduces a few new error cases. A hybrid torrent may have mismatching v1
```

Comment:
It's not in the scope of the bug, but I think this section also ought to mention how `torrent_handle::info_hashes()` can change when metadata is received. It's an important consideration for any code that indexes torrents